eslint.config.mjs

```diff
@@ -1,7 +1,19 @@
-import globals from "globals";
-
+import globals from 'globals';
 
 /** @type {import('eslint').Linter.Config[]} */
 export default [
-  {languageOptions: { globals: globals.browser }},
-];
\ No newline at end of file
+	{
+		files: ['src/**/e2e.js'],
+		ignores: [
+			'**/*.config.js',
+			'!**/eslint.config.js',
+			'node_modules/',
+			'allure-results/',
+			'allure-report',
+			'combined.log',
+		],
+		rules: {
+			semi: 'error',
```

Comment:
Necessary to add more rules. For example, you can put the next one:
'max-len': ['error', { code: 180 }],
    'curly': ['error', 'multi-line'],
    'indent': 'off',
    'yoda': 'error',
    'default-case': 'error',
    'camelcase': ['warn', { ignoreDestructuring: true, properties: 'never' }],
    'no-trailing-spaces': 'off',
    'no-console': 'off',
    'no-undef': 'off',
    'no-sync': 'error',
    'no-var': 'error',
    'no-shadow': 'error',
    'no-nested-ternary': 'error',
    'no-unmodified-loop-condition': 'error',
    'no-unreachable-loop': 'error',
    'no-constant-binary-expression': 'error',
    'no-return-assign': 'error',
    'no-duplicate-imports': 'error',
    // 'no-process-env': 'error',
    'no-bitwise': 'error',
    'no-proto': 'error',
    'no-self-compare': 'error',
    'no-await-in-loop': 'error',
    'no-loop-func': 'error',
    'no-iterator': 'error',
    'no-unused-vars': 'off',
    'no-implicit-coercion': ['error', { allow: ['!!'] }],
    'for-direction': 'error',
    'quotes': ['warn', 'single', { allowTemplateLiterals: true }],
    'guard-for-in': 'error',
    'quote-props': ['warn', 'consistent'],
    'func-style': ['warn', 'expression'],
    'arrow-parens': ['warn', 'as-needed'],
    'dot-notation': 'warn',
    'class-methods-use-this': ['warn'],
    'import/no-extraneous-dependencies': ['error'],
    'import/no-import-module-exports': ['error'],
    '@typescript-eslint/no-use-before-define': 'off',
    '@typescript-eslint/ban-ts-comment': ['error'],
    '@typescript-eslint/no-inferrable-types': 'error',
    '@typescript-eslint/no-explicit-any': 'off',
    '@typescript-eslint/no-namespace': ['error', { allowDeclarations: true }],
    '@typescript-eslint/no-unused-vars': ['error'],
    '@typescript-eslint/explicit-function-return-type': 'off',
    '@typescript-eslint/type-annotation-spacing': 'warn',
    'no-only-tests/no-only-tests': 'error',
    'no-restricted-syntax': [
      'error',
      {
        selector: 'ForInStatement',
        message:
          'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.',
      },
      {
        selector: 'LabeledStatement',
        message: 'Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.',
      },
      {
        selector: 'WithStatement',
        message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
      },
    ],